testapps/test_pillowtop/tests/test_reindexer.py

```diff
@@ -132,22 +133,16 @@ def _assert_index_empty(self, esquery=CaseES()):
         self.assertEqual(0, results.total)
 
 
-class CheckpointCreationTest(TestCase):
+class CheckpointCreationTest(CallCenterDomainMockTest):
     # this class is only here so you can run these explicitly
     pass
 
 
 @generate_cases([
     ('app', 'ApplicationToElasticsearchPillow'),
-    ('case', 'CaseToElasticsearchPillow'),
-    ('form', 'XFormToElasticsearchPillow'),
     ('domain', 'KafkaDomainPillow'),
-    ('user', 'UserPillow'),
-    ('group', 'GroupPillow'),
     ('ledger-v1', 'LedgerToElasticsearchPillow'),
     ('sms', 'SqlSMSPillow'),
-    ('report-case', 'ReportCaseToElasticsearchPillow'),
-    ('report-xform', 'ReportXFormToElasticsearchPillow'),
```

Comment:
@emord 
I am removing these for two reasons, let me know if this doesn't make sense.
1. These reindexers are now using the individual processors instead of the combined pillows
2. The reindexing process assumes that the standalone pillow will need to catch up with combined pillow for which we don't want to reset checkpoints.